### DISCUSSION TITLE
Assembly compilation and .NET Core link

### DISCUSSION THREAD

**Discussion by SebMichaud** [Benji667/LabVIEW_NPOI#11] [Q&A]
Hi Benjamin,

Does the assembly as links with .NET Core ? If yes how does it works with LabVIEW which doesn't support .NET Core ?

Do you use VS to compile the assembly ?
With a .NET framework project ?

Best regards,

Sébastien

**Comment by Benji667** [comment]
Hi @SebMichaud ,

Thanks for your question.

The NPOI 2.1.1 is build up on .NET Framework, that let's LabVIEW the ability to load and use it.

**Reply by SebMichaud** [reply]
OK thanks.
Have you compile it by yourself or was it already compiled ?
Have a good day

**Reply by Benji667** [reply]
Hi SebMichaud,

I used the .NET assembly included within the release 2.1.1 (already compiled).
